NuRadioMC/simulation/simulation.py

```diff
@@ -1397,7 +1401,11 @@ def _write_ouput_file(self):
             if(key.startswith("station_")):
                 continue
             if(not key in fout.keys()):  # only save data sets that havn't been recomputed and saved already
-                fout[key] = np.array(self._fin[key])[saved]
+                if self._fin[key].dtype.char == 'U':
+                    fout[key] = np.array(self._fin[key].astype('S'))[saved]
```

Comment:
why is this explicit casting required?

Reply:
h5py does not the numpy string type. So the best way I could come up with to solve this is to cast it back into a bytestring

Reply:
In the event generator we do it like this
```
            if value.dtype.kind == 'U':
                fout[key] = np.array(value, dtype=h5py.string_dtype(encoding='utf-8'))[start_index:stop_index]
            else:
                fout[key] = value[start_index:stop_index]
```
this might be the betters solution(?). We want the strings to be unicode I guess.

Reply:
Good point.